### PR TITLE
Make daily blocks run sequentially

### PR DESCRIPTION
**Pull request by FrancoGiachetta**
This PR divides the list of blocks in 8 lists of 5 blocks each. It creates 8 workers, each of which depends on the previous to begin execution. It will hopefully avoid both the github and the juno's timeouts. 

## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

**Comment by JulianGCalderon** [comment]
@FrancoGiachetta
Although I haven't tested it, we may be able to use the `max-parallel` option, instead of doing it manually:
- https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymax-parallel

**Comment by FrancoGiachetta** [comment]
Thanks! Didn't know of that one, I'll try it.

**Comment by FrancoGiachetta** [comment]
Done

File: .github/workflows/daily.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ jobs:
       RPC_ENDPOINT_TESTNET: ${{ secrets.RPC_ENDPOINT_TESTNET }}
       RPC_ENDPOINT_MAINNET: ${{ secrets.RPC_ENDPOINT_MAINNET }}
     strategy:
+      max-parallel: 1
```
Comment (JulianGCalderon) [inline .github/workflows/daily.yml:21]: We should discuss which number to pick, If we run everything sequentially, the whole workflow would take a long time (maybe more than a day). I'd increase max-parallel to something like 10.

**Review by JulianGCalderon — CHANGES_REQUESTED** [review]

File: .github/workflows/daily.yml (hunk 1 above)
Comment (FrancoGiachetta) [reply .github/workflows/daily.yml:21]: You are right, silly me. I'll leave it to 10 to discuss it.

**Review by JulianGCalderon — APPROVED** [review]

**Review by edg-l — APPROVED** [review]

**Pull request merged**